src/ingestors/fxhash/index.ts

```diff
@@ -47,13 +47,22 @@ export class FxHashIngestor implements MintIngestor {
     const image = token.metadata?.thumbnailUri || '';
 
     mintBuilder.setName(token.name).setDescription(description).setFeaturedImageUrl(image);
-
-    // The 1 is reserveId in their contract... not clear what for... but it's always 1
+    mintBuilder.setMintOutputContract({ chainId: contract.chainId, address: contract.contractAddress });
+    mintBuilder.setCreator({
+      name: token.author.account.username,
+      imageUrl: token.author.account.profile.picture,
+      walletAddress: token.author.account.wallets?.find((w: any) => w.address.startsWith('0x'))?.address,
+      websiteUrl: token.author.account.profile.website,
+      description: token.author.account.profile.description,
+      twitterUsername: token.author.account.profile.twitter?.split('/').pop(),
+    })
+
+    // The 0 is reserveId in their contract... not clear what for... but 0 seems to always work
     const totalPriceWei = await getFxHashMintPriceInEth(
       contract.chainId,
       contractAddress,
       contract.contractAddress,
-      1,
+      0,
```

Comment:
FYI @yungwkndllc I have no idea what this field is, but... it sometimes works with 1, but in all cases I could find it works with 0 when it works with 1, or when it won't work with 1.

So favoring 0 🤷